component-playground/lib/get-component-fixture-tree.js

```diff
@@ -20,5 +22,19 @@ module.exports = function() {
     fixtures[componentName].fixtures[fixtureName] = requireFixture(fixturePath);
   });
 
+  requireComponent.keys().forEach(function(componentPath) {
+    var pathParts =  parseComponentPath(componentPath),
+      componentName = pathParts[1];
```

Comment:
The indentation here is wrong as well, see https://github.com/skidding/cosmos/pull/136/files#r32058142